packages/sights/src/lib/types.ts

```diff
@@ -86,6 +112,10 @@ export interface Sight {
    * The vehicle type used in the sight overlay.
    */
   vehicleType: VehicleType;
+  /**
+   * The name of the task.
```

Comment:
Replace this with `The list of tasks that must be run on this sight`

Reply:
Resolved.